README.rst

```diff
@@ -360,6 +360,10 @@ Choose from the list of available fixers:
                         Remove trailing commas in list
                         function calls.
 
+* **lower_case_cast** [symfony]
```

Comment:
my proposal was plural because of `lowercase_constants` and `lowercase_keywords`. But I didn't look at other fixer names. 
So I'm not sure here, too ;)